lib/camp.js

```diff
@@ -745,25 +745,45 @@ function Path(matcher) {
 }
 
 // Path unit.
-function pathLikeUnit(serverMethod, statusCode) {
-  statusCode = statusCode || 200;
+function pathLikeUnit(serverMethod, statusCode, httpMethods) {
+  httpMethods = httpMethods || [];
   return function unit(server) {
```

Comment:
Not that important, but could you rename this function to `pathLikeUnit`? It eases debugging.